src/styles/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  display: flex;
+  font-size: 12px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 $shadow-color;
+
+  padding: 0 50px;
+
+  &__logo {
+    display: flex;
+  }
+
+  .navigation {
```

Comment:
why are you doing nesting here?
Would be better if you move each block to separate file